src/styles/utilities/__tests__/color.test.js

```diff
@@ -1,5 +1,5 @@
 import colors from '../../configs/colors'
-import { getColor, getThemeBrandProp } from '../color'
+import { getColor, getThemeBrandProp, rgba } from '../color'
```

Comment:
Not that it really matters, but doesn't look like we're using the `rgba` method in this test, right?

Reply:
Oops! You're right. I originally had the tests here. Will remove!! Thanks 😍